lib/config.coffee

```diff
@@ -98,3 +98,8 @@ module.exports =
         on_the_fly:
           type: 'boolean'
           default: true
+    locale:
+      type: 'string'
+      title: 'Locale when running PROS CLI'
+      description: 'If the default value isn\'t working, try C.UTF-8 or blank to possibly inherit.'

```

Comment:
> [...]or blank to possibly inherit.

I'd say
> [...]or leave this field blank to possibly inherit from the system environment.

If possible, it might be nice to link to [this Atom issue](https://github.com/atom/atom/issues/4126) if people want to know more about it.